ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -169,22 +173,30 @@ class LeftNavMenuViewController: UIViewController {
             menuAction()
         }
 
-        var statusCellChildren: [MSFListCellState] = []
+        let menuSection = leftNavMenuList.state.createSection()
+
+        statusCell = menuSection.createCell()
+        guard let statusCell = statusCell else {
```

Comment:
You don't need the guard here because you're creating the state variable right above and it won't be nil.
You just need to assign it to the property so that the setPresence func has access to it.